sdk/python/pulumi_kubernetes/helm/v4/_inputs.py

```diff
@@ -212,5 +212,3 @@ def username(self) -> Optional[pulumi.Input[str]]:
     @username.setter
     def username(self, value: Optional[pulumi.Input[str]]):
         pulumi.set(self, "username", value)
-
```

Comment:
I suggest removing non-essential whitespace changes since your fix is only about the typo.
(In some cases trailing empty lines can event be meaningful!)

Reply:
This is generated code, you can ignore it.